experiments/modern/src/maki-interpreter/variable.js

```diff
@@ -16,10 +17,11 @@ class Variable {
     if (this._unsubscribeFromValue != null) {
       this._unsubscribeFromValue();
     }
-    if (this.typeName === "OBJECT") {
+    if (this.global && this.typeName === "OBJECT" && value && value.js_listenToAll) {
```

Comment:
I'd prefer an explicit nullish check over a "truthy" check for `value`: `&& value != null`. I should add a lint rule for this if I _really_ care so much.

Don't all maki objects extend from `MakiObject` and thus have `.js_listenToAll`?

Reply:
Sounds good. I found I needed to add this when I did a `findObject` that didn't return a value, so a time we are expecting a `MakiObject` but get `null`.  Maybe its just an `instanceof` check instead of null?